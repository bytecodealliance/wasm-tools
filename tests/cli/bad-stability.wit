// FAIL: component wit %

package a:b;

world c {
  @since(version = 0.2.0)
  import d:e/f-g-h@0.2.3;
}

package d:e@0.2.3 {
  interface f-g-h {}
}
