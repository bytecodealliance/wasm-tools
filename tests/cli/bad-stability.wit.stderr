error: failed to process world item in `c`

Caused by:
    0: package [a:b] contains a feature gate with a version specifier, so it must have a version
            --> tests/cli/bad-stability.wit:7:14
             |
           7 |   import d:e/f-g-h@0.2.3;
             |              ^----
